packages/react-components/src/paragraph/src/Paragraph.jsx

```diff
@@ -42,7 +46,8 @@ export function InnerParagraph(props) {
                     size,
                     className: cssModule(
                         "o-ui-p",
-                        normalizeSize(size)
+                        normalizeSize(size),
+                        "o-ui-text-color-inherit"
```

Comment:
3 things:

1- How does paragraph `inherit` and `color: "inherit"` props cope together?
2- You don't even use the color prop you added since you always render the "o-ui-text-color-inheri" class instead.